 public Exception getContainedException() { return containedException; } 